content/docs/installation/compatibility.md

```diff
@@ -55,6 +55,35 @@ out version 1.21 for Kubernetes masters. Installation via the helm chart
 may end in an error message but cert-manager is reported to be working by
 some users. Feedback and PRs are welcome.
 
+**Problem**: GKE Autopilot does not allow modifications to the `kube-system`-namespace.
+
+Historically we've used the `kube-system` namespace to prevent multiple installations of cert-manager in the same cluster.
+
+Installing cert-manager in these environments with default configuration can cause issues with bootstrapping.
+Some signals are:
+
+* `cert-manager-cainjector` logging errors like:
+
+```text
+E0425 09:04:01.520150       1 leaderelection.go:334] error initially creating leader election record: leases.coordination.k8s.io is forbidden: User "system:serviceaccount:cert-manager:cert-manager-cainjector" cannot create resource "leases" in API group "coordination.k8s.io" in the namespace "kube-system": GKEAutopilot authz: the namespace "kube-system" is managed and the request's verb "create" is denied
+```
+
+* `cert-manager-startupapicheck` not completing and logging messages like:
+
+```text
+Not ready: the cert-manager webhook CA bundle is not injected yet
+```
+
+**Solution**: Configure cert-manager to use a different namespace for leader election, like this:
+
+```console
+helm install \
+  cert-manager jetstack/cert-manager \
+  --namespace cert-manager \
+  --create-namespace \
+  --version v1.8.0 --set global.leaderElection.namespace=cert-manager
```

Comment:
suggestion: it scares me a little to add another place where we need to update the cert-manager version when a new version is released. It's already a bit of a messy manual job.

In this use case would it be OK to use `v<version-name>` instead of `v1.8.0`?

(this kind of thing should be automated for `/docs` but I don't think we're there yet)

Reply:
Good point! I'll put a placeholder there